archive/archive_asset_lookup/src/endpoint.py

```diff
@@ -0,0 +1,103 @@
+import os
+import tempfile
+import logging
+import boto3
+import botocore
+import settings
+from boto3.dynamodb.conditions import Key
+
+
+def lambda_handler(event, context):
```

Comment:
The aws-utils package has some helpers for lambdas that might be worth a look (see the snapshot_scheduler lambda).

Reply:
we're just changing this to use a `main` function so it works with the lambda terraform module properly. will check it out!